app/controllers/api/v1x0/portfolios_controller.rb

```diff
@@ -102,6 +103,21 @@ def portfolio_params
       def portfolio_copy_params
         params.permit(:portfolio_id, :portfolio_name)
       end
+
+      def permissions
```

Comment:
This method should be moved to a service along with the supporting method `verify_permissions` so we can keep with our slim controllers plan.

Reply:
@syncrou I have have moved it to lib/rbac/permission.rb